nuxt.config.ts

```diff
@@ -1,26 +1,79 @@
 const noscriptHtml = `
-<section class="section">
-<div class="container">
-  <h1 class="title is-1">Arctic Data Collaborative</h1>
-  <div class="content is-size-5">
-    <p>The Arctic Data Collaborative (ARDAC) provides highly-curated,
-        relevant <b>Arctic datasets</b>, streamlined <b>data access</b>,
-        processing <code>code examples</code> and <b>visualizations</b> to
-        support research in the North.
-    </p>
-    <p>
-      ⚠️ We&rsquo;re sorry, but this web tool requires
-      Javascript to be enabled to run. <strong>
-        Please email us at
-        uaf-snap-data-tools@alaska.edu
-      </strong> if you would like assistance to
-      access content on this site.
-    </p>
-  </div>
-</div>
+<style>
+  section {
+    width: 600px;
+    margin: 50px auto;
+    font-size: 110%;
+    font-family: sans-serif;
+    line-height: 1.3;
+  }
+  li {
+    margin-bottom: 0.25rem;
+  }
+</style>
+<section>
+  <h1>Arctic Data Collaborative</h1>
+
+  <p>
+    The Arctic Data Collaborative (ARDAC) provides highly-curated, relevant
+    <b>Arctic datasets</b>, streamlined <b>data access</b>, processing
+    <code>code examples</code> and <b>visualizations</b> to support research in
+    the North.
+  </p>
+  <p>
+    ⚠️ We&rsquo;re sorry, but this web tool requires Javascript to be enabled to
+    run. <strong>Please email us at uaf-snap-data-tools@alaska.edu</strong> if
+    you would like assistance to access content on this site.
+  </p>
+
+  <h3>Content available on this site</h3>
+  <p>
+    This tool has a wide arrange of best-available climate science datasets and
+    supporting content. Datasets include:
+  </p>
+  <ul>
+    <li>
+      Temperature and precipitation, from both CMIP5 and CMIP6 climate models,
+      at 2km resolution over Alaska and western Canada
+    </li>
+    <li>
+      Derived climate indicators, such as Wet Days Per Year and Deep Winter
+      Days, provide additional meaning and context for changing climate
```

Comment:
```suggestion
      Derived climate indicators, such as we days per year and deep winter
      days, provide additional meaning and context for changing climate
```

The datasets mentioned elsewhere in the noscript text are not capitalized, so this makes these consistent.